date.lua

```diff
@@ -10,6 +10,12 @@ includedirs {
   "include",
 }
 
+defines{
+"__ORDER_BIG_ENDIAN__=0",
```

Comment:
Lacking these triggers a failure to build in my MSVC build.

I could move these to the Windows platform if thats prefereable?

Reply:
Ahh I see. I think so since clang seems to have these built-in with different values so I'm not sure what effects that would have. Where are these values defined in the upstream project for Windows?

Reply:
Apologies, I'm not familiar with what "Upstream" means in this context. 

From what I cant tell MSVC doesn't easily provide a way to access Endian data. The values I use are just to force HH to compile for little endian-ness. We can likely use the numbers provided from Clang, if that answers the question at all 😄

Reply:
Ahh that's right. There's no "upstream" way to build this because the original repo (upstream) this is forked off of doesn't even use these codepaths at all on Windows.  Better to be consistent with the clang values then. Can you also put a comment on why these need to be defined on Windows.